### PR TITLE
Fix spelling errors in documentation

### PR DESCRIPTION
**Pull request by emilysunaryo**
## Description
In this PR, I edited 3 typos in the pushdown.md documents. 

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/optimizer/pushdown.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ typically resulting from a condition in a `WHERE` clause to omit unnecessary
 rows. The processing is pushed down to the data source by the connector and then
 processed by the data source.
 
-If predicate pushdown for a specific clause is succesful, the `EXPLAIN` plan
+If predicate pushdown for a specific clause is successful, the `EXPLAIN` plan
```
Comment (ebyhr) [inline docs/src/main/sphinx/optimizer/pushdown.md:27]: Could you fix the same typo in `json.md`?

**Review by ebyhr — APPROVED** [review]

**Comment by mosabua** [comment]
Please fix commit message to follow https://trino.io/development/process#contribution-process .. specifically https://cbea.ms/git-commit/

Once CLA is processed we can merge.

**Comment by ebyhr** [comment]
> Please fix commit message

The current commit looks good to me. What's your suggestion?

**Comment by mosabua** [comment]
> > Please fix commit message
> 
> The current commit title looks good to me. What's your suggestion?

Typically commit message should not talk about what file is affected .. especially also since you might add other files. You could do something like 

`Fix spelling errors in documentation` or `Correct spelling in docs` or `Fix grammar issues in docs` or similar.

File: docs/src/main/sphinx/optimizer/pushdown.md (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/optimizer/pushdown.md:27]: @emilysunaryo ?

Comment (mosabua) [reply docs/src/main/sphinx/optimizer/pushdown.md:27]: You can just do a global search for that typo and fix all of them in this PR

**Comment by mosabua** [comment]
@cla-bot check

**Review by mosabua — APPROVED** [review]
Looks good now. Thank you.

**Pull request merged**